src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -189,6 +189,26 @@ def get_subvolume_path(self, spec):
             raise VolumeException(-e.args[0], e.args[1])
         return path
 
+    def get_dir_entries(self, path):
+        """
+        Get the directory names in a given path
+        :param path: the given path
+        :return: the list of directory names
+        """
+        dirs = []
+        try:
+            with self.fs.opendir(path) as dir_handle:
+                d = self.fs.readdir(dir_handle)
+                while d:
+                    if (d.d_name not in (b".", b"..")) and d.is_dir():
+                        dirs.append(d.d_name)
+                    d = self.fs.readdir(dir_handle)
+        except cephfs.ObjectNotFound:
+            raise VolumeException(-errno.ENOENT, "Path '{0}' not found".format(path))
```

Comment:
I think we should return empty dirs list here instead of VolumeException. @ajarr What do you think?

When subvolumegroups are not created,
```
$ ./bin/ceph fs subvolumegroup ls myfs
Error ENOENT: Path '/volumes' not found
```

Reply:
+1

Reply:
(Add test for this too!)

Reply:
@batrick Fixed your comments. Please check.